python-template/{{cookiecutter.placeholder_repo_name}}/README.rst

```diff
@@ -1,33 +1,33 @@
 {{cookiecutter.project_name}}
 #############################
 
-|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
-|license-badge|
+.. note::
 
-{{ cookiecutter.project_short_description}}
+  This README was auto-generated. Please review its contents and update all
+  relevant sections.
 
-The ``README.rst`` file should start with a brief description of the repository,
-which sets it in the context of other repositories under the ``edx``
-organization. It should make clear where this fits in to the overall edX
-codebase.
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge| |status-badge|
 
-Overview (please modify)
-************************
+Purpose
+*******
 
-The ``README.rst`` file should then provide an overview of the code in this
-repository, including the main components and useful entry points for starting
-to understand the code in more detail.
+{{ cookiecutter.project_short_description}}
 
-Documentation
-*************
+The ``README.rst`` file should start with a brief description of the repository and its purpose.
```

Comment:
I know this is merged already, but thinking about linting, and also for clarification: what if we started each paragraph like this with `PLACEHOLDER:`, then linted for the appearance of that word in READMEs?  It would help everyone (maintainer and readers of the file) understand that the paragraph is just a placeholder, leave a strong signal that something needs to be done, and be easily lintable.

Reply:
Actually, I will make a PR with this change, and we can discuss there.

Reply:
https://github.com/openedx/edx-cookiecutters/pull/244